src/map/pc.c

```diff
@@ -4392,10 +4392,11 @@ int pc_payzeny(struct map_session_data *sd,int zeny, enum e_log_pick_type type,
 /*==========================================
  * Cash Shop
  *------------------------------------------*/
-
+//Changed Kafrapoints calculation. [Normynator]
 int pc_paycash(struct map_session_data *sd, int price, int points)
 {
 	int cash;
+	int smp;
```

Comment:
what's smp? Use better name, so others who might be looking won't have to check what it does, but would come to know from name of variable itself.